CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### 🛡 Security
 
+- [CVE-2022-25758]Bump scss-tokenizer from 0.3.0 to 0.4.3 ([#3727](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3727))
```

Comment:
🤣 If I'm wrong about whether we prefer a space or not, feel free to ignore and throw a shoe at me

```suggestion
- [CVE-2022-25758] Bump scss-tokenizer from 0.3.0 to 0.4.3 ([#3727](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3727))
```

Reply:
Don't worry, we don't condone violence in the workplace.